locales-pending/settings-premium.ftl

```diff
@@ -22,7 +22,9 @@ settings-cancel-plus-step-survey-note = Please note, all of your { -brand-monito
 settings-cancel-plus-step-survey-cta-label = Continue to cancellation
 settings-cancel-plus-step-survey-cancel-label = Never mind, take me back
 settings-cancel-plus-step-redirecting-heading = We appreciate your feedback
-settings-cancel-plus-step-redirecting-lead = We’ll automatically redirect you to your { -brand-mozilla-account } where you can cancel your { -brand-monitor-plus } subscription.
+settings-unsubscribe-dialog-confirmation-redirect-title = Directing you to your { -brand-mozilla } account to cancel
```

Comment:
```suggestion
settings-unsubscribe-dialog-confirmation-redirect-title = Directing you to your { -brand-mozilla-account } to cancel
```

All the instances of `{ -brand-mozilla } account`  are mistakes that should be fixed over time.